bridges/DealabsBridge.php

```diff
@@ -458,18 +551,38 @@ private function relativeDateToTimestamp($str) {
 		return $date->getTimestamp();
 	}
 
+	/**
+	 * Returns the RSS Feed title according to the parameters
+	 * @return string the RSS feed Tiyle
+	 */
 	public function getName(){
 		switch($this->queriedContext) {
-			case 'Recherche par Mot(s) clé(s)':
-				return self::NAME . ' - Recherche : '. $this->getInput('q');
+			case $this->i8n('context-keyword'):
+				return $this->i8n('bridge-name') . ' - '. $this->i8n('title-keyword') .' : '. $this->getInput('q');
 				break;
-			case 'Deals par groupe':
-				$values = self::PARAMETERS['Deals par groupe']['groupe']['values'];
-				$groupe = array_search($this->getInput('groupe'), $values);
-				return self::NAME . ' - Groupe : '. $groupe;
+			case $this->i8n('context-group'):
+				$values = static::PARAMETERS[$this->i8n('context-group')]['group']['values'];
```

Comment:
Why not using getParameter here ?

Reply:
Because I didn't know it existed ! As I know it now, I fixed it !